src/components/Cluster/ClusterDetail/NodePool.js

```diff
@@ -119,7 +121,27 @@ class NodePool extends Component {
         {/* Hide the rest of fields when editing name */}
         {!isNameBeingEdited && (
           <>
-            <Code>{node_spec.aws.instance_type}</Code>
+            <Code>
+              {node_spec.aws.use_alike_instance_types
+                ? 'mixed'
+                : node_spec.aws.instance_type}
+            </Code>
+            <NodesWrapper>
+              {instance_distribution &&
+              typeof instance_distribution.on_demand_base_capacity !==
+                'undefined'
+                ? instance_distribution.on_demand_base_capacity
+                : '-'}
+              /
+              {instance_distribution &&
+              typeof instance_distribution.on_demand_percentage_above_base_capacity !==
+                'undefined'
+                ? `${
+                    100 /* eslint-disable-line no-magic-numbers */ -
+                    instance_distribution.on_demand_percentage_above_base_capacity
+                  } %`
+                : '-'}
```

Comment:
Can you move this to a separate function, please?

Reply:
check out 2a3942e